src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -253,6 +253,19 @@ conversion, so type inference fails because the type is not unique. Please note
 that you must write the `(())` in one sequence without intermediate whitespace
 so that rustdoc understands you want an implicit `Result`-returning function.
 
+As of version 1.37.0, this simplification also works with `Option`s, which can
+be handy to test e.g. iterators or checked arithmetic, for example:
+
+```ignore
+/// ```
+/// let _ = &[].iter().next()?;
+///# Some(())
```

Comment:
Also, this line should have added a space between `/// #`, since historically having single lines without that leading space has messed up rustdoc's Markdown parser.

Reply:
ping @llogiq